http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/clients/HealthClientServiceBuilder.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.ts.openshift.http.clients;
+
+import java.io.IOException;
+import java.net.URI;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+
+public class HealthClientServiceBuilder  extends CommonClientService {
+
+    public static HealthClientServiceBuilder build(String uri) {
+        return new HealthClientServiceBuilder(uri);
+    }
+
+    private HealthClientServiceBuilder(String uri) {this.uri = uri;}
```

Comment:
The uri field belongs to CommonClientService, so CommonClientService should have a constructor with this field instead.